contracts/PvpArena.sol

```diff
@@ -78,6 +78,10 @@ contract PvpArena is Initializable, AccessControlUpgradeable {
     mapping(uint256 => Fighter) public fighterByCharacter;
     /// @dev Active duel by characterID currently attacking
     mapping(uint256 => Duel) public duelByAttacker;
+    /// @dev ranking points by character
+    mapping(uint256 => uint256) public characterRankingPoints;
+    /// @dev funds available for withdrawal by address
+    mapping(address => uint256) public withdrawableFundsByAddress;
```

Comment:
We have other funds that can be withdrawn too `_duelEarningsByCharacter`. So I'd say we name it something similar. 

About visibility, it might be better to have funds private unless we want to expose them more easily, though I'm not too sure if there's any difference here tbh. But let's at least be consistent